nouns.json

```diff
@@ -488,6 +488,168 @@
+        "EU": { "atom": "", "pstn": "" },
+        "FA": { "atom": "", "pstn": "" },
+        "FI": { "atom": "", "pstn": "" },
+        "FR": { "atom": "", "pstn": "" },
+        "IT": { "atom": "", "pstn": "" },
+        "JA": { "atom": "", "pstn": "" },
+        "KO": { "atom": "", "pstn": "" },
+        "PL": { "atom": "", "pstn": "" },
+        "PT": { "atom": "", "pstn": "" },
+        "RU": { "atom": "", "pstn": "" },
+        "TR": { "atom": "", "pstn": "" },
+        "ZH": { "atom": "", "pstn": "" }
+      },
+      "021e011b": {
+        "CA": { "atom": "de Lie", "pstn": "after" },
+        "DE": { "atom": "", "pstn": "" },
+        "EN": { "atom": "Lie", "pstn": "before" },
+        "ES": { "atom": "de Lie", "pstn": "after" },
+        "EU": { "atom": "", "pstn": "" },
+        "FA": { "atom": "", "pstn": "" },
+        "FI": { "atom": "", "pstn": "" },
+        "FR": { "atom": "", "pstn": "" },
+        "IT": { "atom": "", "pstn": "" },
+        "JA": { "atom": "", "pstn": "" },
+        "KO": { "atom": "", "pstn": "" },
+        "PL": { "atom": "", "pstn": "" },
+        "PT": { "atom": "", "pstn": "" },
+        "RU": { "atom": "", "pstn": "" },
+        "TR": { "atom": "", "pstn": "" },
+        "ZH": { "atom": "", "pstn": "" }
+      },
+      "0f7c0373": {
+        "CA": { "atom": "exterior", "pstn": "after" },
+        "DE": { "atom": "", "pstn": "" },
+        "EN": { "atom": "exterior", "pstn": "before" },
+        "ES": { "atom": "exterior", "pstn": "after" },
+        "EU": { "atom": "", "pstn": "" },
+        "FA": { "atom": "", "pstn": "" },
+        "FI": { "atom": "", "pstn": "" },
+        "FR": { "atom": "", "pstn": "" },
+        "IT": { "atom": "", "pstn": "" },
+        "JA": { "atom": "", "pstn": "" },
+        "KO": { "atom": "", "pstn": "" },
+        "PL": { "atom": "", "pstn": "" },
+        "PT": { "atom": "", "pstn": "" },
+        "RU": { "atom": "", "pstn": "" },
+        "TR": { "atom": "", "pstn": "" },
+        "ZH": { "atom": "", "pstn": "" }
+      },
+      "08e90296": {
+        "CA": { "atom": "matricial", "pstn": "after" },
+        "DE": { "atom": "", "pstn": "" },
+        "EN": { "atom": "matrix", "pstn": "before" },
+        "ES": { "atom": "matricial", "pstn": "after" },
+        "EU": { "atom": "", "pstn": "" },
+        "FA": { "atom": "", "pstn": "" },
+        "FI": { "atom": "", "pstn": "" },
+        "FR": { "atom": "", "pstn": "" },
+        "IT": { "atom": "", "pstn": "" },
+        "JA": { "atom": "", "pstn": "" },
+        "KO": { "atom": "", "pstn": "" },
+        "PL": { "atom": "", "pstn": "" },
+        "PT": { "atom": "", "pstn": "" },
+        "RU": { "atom": "", "pstn": "" },
+        "TR": { "atom": "", "pstn": "" },
+        "ZH": { "atom": "", "pstn": "" }
+      },
+      "1ca304ab": {
+        "CA": { "atom": "simètrica", "pstn": "after" },
+        "DE": { "atom": "", "pstn": "" },
+        "EN": { "atom": "symmetrical", "pstn": "before" },
```

Comment:
it seems that "symmetric" is much more common that "symmetrical" (though I had to do a google search to be sure...), so could you change this one too please? (the new id would be `13b903de`)

Reply:
No problem with this one. Done.